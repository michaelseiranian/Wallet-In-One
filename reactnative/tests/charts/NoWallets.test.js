// HomePage.test.js
import React from 'react';
import NoWallets from '../../screens/charts/chartComponents/noWallets';
import renderer from 'react-test-renderer';

describe('<NoWallets />', () => {
    it('snapshot test', () => {
        const snapshot = renderer.create(<NoWallets />);
        expect(snapshot).toMatchSnapshot();         
      });
  });