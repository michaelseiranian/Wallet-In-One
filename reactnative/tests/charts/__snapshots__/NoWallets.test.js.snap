// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoWallets /> snapshot test 1`] = `
<RCTScrollView
  contentContainerStyle={
    {
      "alignItems": "center",
      "backgroundColor": "white",
      "flexGrow": 1,
      "justifyContent": "center",
      "paddingBottom": 20,
    }
  }
>
  <View>
    <Text
      style={
        [
          {
            "color": "black",
          },
          {
            "textAlign": "center",
          },
        ]
      }
    >
      Welcome to Wallet-In-One!
      

      Connect your Assets to See your Funds
    </Text>
  </View>
</RCTScrollView>
`;
